exonum-java-binding-common/src/main/java/com/exonum/binding/common/proofs/map/CheckedMapProof.java

```diff
@@ -33,7 +33,7 @@
  * // Convert to checked
  * CheckedMapProof checkedProof = proof.check();
  * // Check the root hash
- * if (checkedProof.compareWithRootHash(expectedRootHash)) {
+ * if (checkedProof.isValid() && checkedProof.compareWithRootHash(expectedRootHash)) {
```

Comment:
Haven't we removed `compareWithRootHash`?

Reply:
We did in `CheckedListProof`, not in `CheckedMapProof`. I can do it in this PR, if we want to remove it?

Reply:
:+1: